libraries/Microsoft.Bot.Builder.Dialogs.Debugging/DataModels/JValueDataModel.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Bot.Builder.Dialogs.Debugging
+{
+    public sealed class JValueDataModel : IDataModel
```

Comment:
>JValueDataModel [](start = 24, length = 15)

really similar to ScalarDataModel - I thought I might need to customize ToString more